assets/js/amp-editor-blocks.js

```diff
@@ -314,6 +345,77 @@ var ampEditorBlocks = ( function() {
 		);
 	};
 
+	/**
+	 * Setup inspector controls for text blocks.
+	 *
+	 * @param {Object} props Props.
+	 * @return {Object|Element|*|{$$typeof, type, key, ref, props, _owner}} Inspector Controls.
+	 */
+	component.setUpTextBlocksInspectorControls = function setUpInspectorControls( props ) {
+		var ampFitText = props.attributes.ampFitText,
+			minFont = props.attributes.minFont,
+			maxFont = props.attributes.maxFont,
+			height = props.attributes.height,
+			isSelected = props.isSelected,
+			el = wp.element.createElement,
+			InspectorControls = wp.blocks.InspectorControls,
+			TextControl = wp.components.TextControl,
+			ToggleControl = wp.components.ToggleControl,
+			PanelBody = wp.components.PanelBody,
+			label = 'Use AMP Fit Text';
+
+		if ( ampFitText ) {
+			return isSelected && (
+				el( InspectorControls, { key: 'inspector' },
+					el( PanelBody, { title: component.data.ampSettingsLabel },
+						el( ToggleControl, {
+							label: label,
+							checked: ampFitText,
+							onChange: function() {
+								props.setAttributes( { ampFitText: ! ampFitText } );
```

Comment:
Can this also result in the `font-size-picker` and `range-control` under Text Settings to be hidden when `ampFitText` is true?